### ISSUE TITLE
model with batchnorm runs overfit in multigpu training,but is ok in single gpu. mutilgpu model is based on cifar10_multi_gpu_train.py

### ISSUE THREAD

**Issue by u4lr451** [tensorflow/tensorflow#3367]
1. mutil gpu code based on cifar10_multi_gpu_train.py(https://github.com/tensorflow/tensorflow/blob/d42facc3cc9611f0c9722c81551a7404a0bd3f6b/tensorflow/models/image/cifar10/cifar10_multi_gpu_train.py)
2. two implementation of batch norm  is overfit in multigpu ,but is ok in sigle gpu.<br>
   one implementation based on inception batch_norm function: (https://github.com/tensorflow/models/blob/master/inception/inception/slim/ops.py)
<br>
   the other implementation base on :
https://codegists.com/snippet/python/batchnormpy_thouis_python

**Comment by concretevitamin** [comment]
This does not sound like a bug in TensorFlow.  Maybe StackOverflow is a better venue?  Although, please do re-open if you've isolated more down to a TF component/op.
